app/templates/components/sl-drop-option.hbs

```diff
@@ -1,7 +1,7 @@
 {{#if label}}
     <a {{action "click" action}} role="menuitem" tab-index="-1">
         {{#if icon}}
-            <img {{bind-attr src=icon}}/>
+            <img {{bind-attr src=icon}} />
```

Comment:
What was the motivation for this change?  Seeing if we need to also change https://github.com/softlayer/sl-ember-components/blob/7ea30420ec7fcbfabcd43e366630ac23d1abb0f9/app/templates/components/sl-grid-header-settings.hbs#L14